### PR TITLE
Workspace plugin downloadBundle improvement

### PR DESCRIPTION
**Pull request by lawrence-lee**

### CODE REVIEW

**Comment by lawrence-lee** [comment]
ci:reopen

**Comment by lawrence-lee** [comment]
ci:retest

File: modules/sdk/gradle-plugins-workspace/src/gradleTest/sourceEqualsDestCheck/build.gradle [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+runGradleTest {
+	doLast {
+		try {
+			tasks.downloadBundle.execute()
+		}
+		catch (Exception e) {
+			assert downloadBundle.state.didWork == true
+			assert downloadBundle.state.executed == true
+			assert e.message.contains("Execution failed for task ':downloadBundle'")
```
Comment (gamerson) [inline modules/sdk/gradle-plugins-workspace/src/gradleTest/sourceEqualsDestCheck/build.gradle]: Lets make sure the exception is the same one that we threw containing the same message.  Its hard to see here if the exception is actually our exception.

Comment (gamerson) [reply modules/sdk/gradle-plugins-workspace/src/gradleTest/sourceEqualsDestCheck/build.gradle]: try `assert e.cause.message.contains("...")`

**Comment by gamerson** [comment]
ci:retest

**Comment by gamerson** [comment]
Pull request submitted to https://github.com/Ithildir/liferay-portal/pull/787. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**